.github/issue_template.md

```diff
@@ -0,0 +1,17 @@
+**Important:** GitHub issues are for feature requests or bug reports. The Capistrano team recommends you use [Stack Overflow](http://stackoverflow.com/questions/tagged/capistrano) for general questions. For more details, please see our [contribution policy](https://github.com/capistrano/capistrano/blob/master/CONTRIBUTING.md).
+
+---
+
+### Steps to reproduce
+If at all possible, please try to create a new Capistrano example project which reproduces the issue you are seeing. You can then upload the individual files to a GitHub Gist or a GitHub project. Often times, an issue is resolved simply by making this test case.
```

Comment:
I understand the concern… I'm mostly thinking of the situation in https://github.com/capistrano/capistrano/issues/1748 where I was able to create https://gist.github.com/will-in-wi/527327e31af30b3eae2068e2965be05b which was the skeleton of a reproducible case.

Perhaps weakening the language would help, e.g. "if at all possible" should probably be "if reasonable" or "if you are running into an issue in Capistrano itself".

Does it make sense what I'm getting at? I'm not sure what the right approach is.